extensions/quarkiverse-systemd-notify.yaml

```diff
@@ -2,6 +2,7 @@
 group-id: "io.quarkiverse.systemd.notify"
 artifact-id: "quarkus-systemd-notify"
 versions:
-- "0.2.0"
-- "0.1.0"
+  - "0.2.0":
+      compatible-with-quarkus-core:
+        - "2.10.0.Final"
```

Comment:
Actually we still haven't implemented minimum version comparison in the compatibility list here. 

In your case I think the best is to exclude the 0.1.0 version (by moving 0.1.0 to the exclude-versions) and mark it as compatible with 2.14.3.Final

Reply:
so, the file should be like this, right?

```
---
group-id: "io.quarkiverse.systemd.notify"
artifact-id: "quarkus-systemd-notify"
versions:
- "0.2.0"
    compatible-with-quarkus-core:
        - "2.14.3.Final"
exclude-versions: ["0.1.0"]
```

Reply:
Yeap, that looks good